<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="RavikumarPC" failures="1" tests="2" name="Erro Validation test" time="34.675" errors="0" timestamp="2023-02-22T22:32:24 IST">
  <testcase classname="ravikumaracademy.tests.ErrorValidationtest" name="loginErrorValidation" time="0.944"/>
  <testcase classname="ravikumaracademy.tests.ErrorValidationtest" name="productErrorValidation" time="26.277">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at ravikumaracademy.tests.ErrorValidationtest.productErrorValidation(ErrorValidationtest.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- productErrorValidation -->
</testsuite> <!-- Erro Validation test -->
